File: executable/r/axes-reversed.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

data <- list(
  list(
    x = c(1, 2), 
    y = c(1, 2), 
    type = "scatter"
  )
)
layout <- list(xaxis = list(autorange = "reversed"))

response <- p$plotly(data, kwargs=list(layout=layout, filename="axes-reversed", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/julia/axes-reversed.jl
using Plotly

using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

data = [
  [
    "x" => [1, 2], 
    "y" => [1, 2], 
    "type" => "scatter"
  ]
]
layout = ["xaxis" => ["autorange" => "reversed"]]

response = Plotly.plot([data], ["layout" => layout, "filename" => "axes-reversed", "fileopt" => "overwrite"])
plot_url = response["url"]